tcpdump.c

```diff
@@ -2372,6 +2398,7 @@ static void
 dump_packet_and_trunc(u_char *user, const struct pcap_pkthdr *h, const u_char *sp)
 {
 	struct dump_info *dump_info;
+    int dlt;
```

Comment:
Thanks! Will do. Please let me know if there's anything else I need to change. (Amcurrently  writing an email to the mailing list explaining this pull request.)